From 45222964fe5632e551713ee9c21fbbff19344877 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Wed, 5 Jul 2017 15:47:15 +0800
Subject: [PATCH 2054/5242] MLK-15342-2 arm64: dts: freescale: imx8mq: add cpu
 opp table

commit  1057cf9d7e12e7ef7396c5af0de0f5675a0c0479 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add i.MX8MQ cpu OPP table info for cpu-freq driver.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi |   14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
index be6d5f5..c0a88e5 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mq.dtsi
@@ -744,3 +744,17 @@
 		status = "disabled";
 	};
 };
+
+&A53_0 {
+	operating-points = <
+		/* kHz    uV */
+		1200000 900000
+		800000	900000
+	>;
+	clocks = <&clk IMX8MQ_CLK_A53_DIV>, <&clk IMX8MQ_CLK_A53_SRC>,
+		<&clk IMX8MQ_ARM_PLL>, <&clk IMX8MQ_ARM_PLL_OUT>,
+		<&clk IMX8MQ_SYS1_PLL_800M>;
+	clock-names = "a53", "arm_a53_src", "arm_pll",
+		"arm_pll_out", "sys1_pll_800m";
+	clock-latency = <61036>;
+};
-- 
1.7.9.5

